x/ibc/03-connection/keeper/handshake.go

```diff
@@ -86,7 +86,7 @@ func (k Keeper) ConnOpenTry(
 
 	ok := k.VerifyMembership(
 		ctx, connection, proofHeight, proofInit,
-		types.ConnectionPath(counterparty.ConnectionID), expConnBz,
+		string(types.KeyPrefixConnection)+types.ConnectionPath(counterparty.ConnectionID), expConnBz,
```

Comment:
This should be abstracted into a helper function

Reply:
yeah, agree. But don't sure the `KeyPrefix` is needed. see https://github.com/cosmos/cosmos-sdk/pull/5313#issuecomment-555541107.